### ISSUE TITLE
Release 2.0?

### ISSUE THREAD

**Issue by jensens** [collective/collective.easyformplugin.registration#5]
It would be great to have a release here (beta or final?)

**Comment by cekk** [comment]
You're right..we are not using it too much right now, so i kinda forgot these new features.

I've added you as maintainer on pypi, so you can manage releases

**Comment by 1letter** [comment]
@jensens @cekk any news here? i would like to fix this #7 , but a new release would be great

**Comment by cekk** [comment]
@1letter let me know if you need it asap or you want to fix #7 before the release. I can release it when you want.

**Comment by jensens** [comment]
I merged #7, release would be great!

**Comment by 1letter** [comment]
@cekk  Thanks for fast response. I agree with @jensens .

**Comment by cekk** [comment]
@1letter what's your username on pypi? I can add you too

**Comment by 1letter** [comment]
@cekk i'm not on pypi. i didn't make a release yet. I don't have the know-how, perhaps.

**Comment by jensens** [comment]
@1letter chance to learn! With zest.releaser it is not that difficult.

**Comment by cekk** [comment]
ops..i've just released it

@1letter for next closed pulls (for example the german one) you can try to release it ;)

If you need help, let me know

**Comment by 1letter** [comment]
@cekk i will learn and test it ;-)
